astropy/utils/console.py

```diff
@@ -631,10 +637,13 @@ def _update_ipython_widget(self, value=None):
         # if none exists.
         if not hasattr(self, '_widget'):
             # Import only if an IPython widget, i.e., widget in iPython NB
-            from IPython.html import widgets
+            try:
+                from ipywidgets import widgets
+            except ImportError:
+                from IPython.html import widgets
             from IPython.display import display
 
-            self._widget = widgets.FloatProgressWidget()
+            self._widget = widgets.FloatProgress()
```

Comment:
`FloatProgressWidget` was deprecated (in favor of `FloatProgress`) in `3.0.0` and removed in `4.0.0`.

`FloatProgress` will not work for `IPython` versions < `3.0`.  If you prefer, I can revert to `FloatProgressWidget` for `2.x`.